src/pages/agile/chickens-versus-pigs/index.md

```diff
@@ -4,21 +4,17 @@ title: Chickens Versus Pigs
 ## Chickens Versus Pigs
 
 "Chickens versus Pigs" refers to a story about a chicken and a pig, where the chicken proposes they open a restaurant called "Ham-n-Eggs".
-The pig refuses, because while the chicken just needs to lay eggs, the pig has more at stake.
+The pig refuses, because while the chicken just needs to lay eggs, the pig has more at stake. **The Chicken is involved, but the Pig is committed!**
 
-In an Agile software development project, the software developer is the pig. If you fail to complete the job, or fail to do it well,
-you have a lot at stake. This could be your reputation, or maybe even your position. Other team members might also be considered pigs, 
-depending on how much they have at stake. 
+In an Agile, the team members are the pigs. They are commited to the product / project they are working on. If the team fails to complete the job, or fails to do it well, they have a lot at stake.
 
-On the other hand, many team members are chickens. For example, the client or a high-level project manager would not really be impacted
-by the failure of the project. They are interested in its success, and might make contributions, but have lower stakes and thus
-have significantly less commitment to the project. 
+On the other hand, the chickens are for example, the client, other teams, or a high-level project manager. They would not really be impacted by the failure of the project directly. They are interested in its success, and might make contributions, but have lower stakes and thus have significantly less commitment to the project. 
 
-You should strive to be a pig rather than a chicken. You can benefit from (but should not rely on) the chickens in order to minimize risk and
-guarantee the project is delivered as efficiently as possible.
+Therefore Agile emphazises on <a href="/agile/self-organization">self-organizing</a> teams. The maxim is to let the commited pigs organize their work by themselves, because they know best. Chickens are only involved and would hardly fail to orginze and improve a working process from the *outside*. 
 
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
 
+<a href="https://www.implementingscrum.com/2006/09/11/the-classic-story-of-the-pig-and-chicken/">Implementing Scrum: The original article and comic</a>
```

Comment:
these links should all be markdown, as per the style guide.
https://github.com/freeCodeCamp/guides#article-style-guide

Reply:
Would like to, but I really don't know how. Find no edit button now. Is it because the PR is closed?

My changes were all about clarification, esp. that it not only developers, but the whole team and about the reason why the story is told: the emphasize on self-organisation.